### PR TITLE
Update README w/ quickstart directions

### PR DESCRIPTION
**Pull request by jayunit100**
Here's an update to the README that shows how to deploy with docker .  thanks to sdurrheimer on #prometheus in irc !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,12 +61,23 @@ Run the production server from the bundled environment:
     bin/env bin/bundle exec bin/thin -p $PORT start
 
 ### Deploy with Docker
+
 To deploy PromDash with Docker, use the [prom/promdash](https://registry.hub.docker.com/u/prom/promdash/).
```
Comment (juliusv) [inline README.md]: While you're editing this, could you change this to:

`...use the [prom/promdash](https://registry.hub.docker.com/u/prom/promdash/) Docker image.` (added words at the end)

File: README.md [hunk 2]
```diff
@@ -61,12 +61,23 @@ Run the production server from the bundled environment:
...
 By default, the image will start the [thin webserver](http://code.macournoyer.com/thin/)
 webserver in production mode. To run rake tasks like migrations, you
-can specify any kind of command as parameter to the Docker image:
+can specify any kind of command as parameter to the Docker image.  
+
+*Super easy quickstart deployment with docker*
```
Comment (juliusv) [inline README.md]: nit: capitalize Docker.

Comment (juliusv) [inline README.md]: Hmm, italic doesn't look like a headline. Maybe `####`?

File: README.md [hunk 3]
```diff
@@ -61,12 +61,23 @@ Run the production server from the bundled environment:
...
+
+A quick start example, with a simple file based local db.
+
+First, we do a database creation, into a shared local volume on our host.
```
Comment (juliusv) [inline README.md]: `First, create the SQLite3 database in a shared local volume on the host:`.

File: README.md [hunk 4]
```diff
@@ -61,12 +61,23 @@ Run the production server from the bundled environment:
...
+
+    docker run -p 3000:3000 -v /tmp/prom:/tmp/prom -e DATABASE_URL=sqlite3:////tmp/prom/file.sqlite3 prom/promdash ./bin/rake db:migrate
+
+Now, we launch prometheus with the database we've just created.
```
Comment (juliusv) [inline README.md]: nit: capitalize Prometheus

File: README.md [hunk 5]
```diff
@@ -61,12 +61,23 @@ Run the production server from the bundled environment:
...
+
 To deploy PromDash with Docker, use the [prom/promdash](https://registry.hub.docker.com/u/prom/promdash/).
 By default, the image will start the [thin webserver](http://code.macournoyer.com/thin/)
 webserver in production mode. To run rake tasks like migrations, you
-can specify any kind of command as parameter to the Docker image:
+can specify any kind of command as parameter to the Docker image.  
+
+*Super easy quickstart deployment with docker*
+
+A quick start example, with a simple file based local db.
```
Comment (juliusv) [inline README.md]: Whole sentence please: `The following is a quick-start example for running PromDash with a file-based local database.`

**Comment by juliusv** [comment]
Thanks!

**Comment by jayunit100** [comment]
okay, updated.

**Comment by juliusv** [comment]
:+1: Thanks a bunch!

**Comment by jayunit100** [comment]
no problem @juliusv  thanks for the help with getting it up and running ! :)

**Pull request merged**